frontend/src/scenes/persons/Person.tsx

```diff
@@ -87,6 +87,21 @@ function PersonCaption({ person }: { person: PersonType }): JSX.Element {
                 <span className="text-muted">First seen:</span>{' '}
                 {person.created_at ? <TZLabel time={person.created_at} /> : 'unknown'}
             </div>
+            <div>
+                <span className="text-muted">Merge restrictions:</span> {person.is_identified ? 'applied' : 'none'}
```

Comment:
Does "applied" mean that this person can't be merged with another person at all?

Reply:
It can be merged with another "none" person by using this person as the distinct_id and "none" person as anon_distinct_id or alias_id. 
It can be merged with another "applied" person only via `$merge_dangerously` events

Reply:
I think there could be a better phrasing for this. We're trying to say that this person cannot be merge into another person right?

A few alternatives that come to mind
`merge possible: false / true`
`merge possible` and `merge restricted`

Could also add ascii checkmarks e.g.
`✓ Merge possible` and `✗ Merge restricted`

Reply:
I don't like `possible` as it's always possible - just you'd need to use `$merge_dangerously`

Reply:
Hmm a few more idea then:

1. `✓ Safe merge possible` : `✗ Dangerous merge only`

2. `✓ Safe merge possible` `✗ Safe merge not possible`

3. `Safe merge possible: ✓` `Safe merge possible: ✗`

I think I like the second option the most

Reply:
I'd want to keep the existing style of the page - so there's a keyword next to "IDs", "First seen" and then the value for it. I also wouldn't want to call it safe merge - that's a new term we don't use anywhere else. Also you can do a safe merge into this person, but you can't merge this person into anyone else.

Reply:
So taking a step back here, here is my understanding of what we're trying to achieve (please correct me if Im wrong)

* We want to communicate in the app if a user has been identified or not, so that our users will better understand if they can safe merge or not

With that in mind, should we do the following:

For anonymous users, instead of:
label: `Merge restrictions: none`
tooltip `Can be used as `$anon_distinct_id` or `alias_id`, click for more info.`

Change this to:
label: `Anonymous user` (or unidentified user)
tooltip `This user has not been identified. Click for more details` and it links to https://posthog.com/docs/data/identify

And then for identified users, change it to:
label: `Identified user`
tooltip `This user has been identified. If this is a duplicate of another user, click for more details on how to merge duplicate users` and it links to https://posthog.com/docs/data/identify#how-to-handle-duplicates-of-the-same-user

What do you think?

Reply:
> We want to communicate in the app if a user has been identified or not, so that our users will better understand if they can safe merge or not

couple of goals:
1. For folks who are not merging users it shouldn't be confusing - "Anonymous user" to me feels confusing there, e.g. if you use backend only and haven't merged with anyone.
2. For folks who are trying to understand the ingestion warning or why their merge failed - they'd understand what was the problem
3. For folks who want to merge this user with someone they will know how to do it

Reply:
> , e.g. if you use backend only and haven't merged with anyone

Users on the backend are identified users, no? Is it possible to have anonymous users on the backend? How? (I thought you need to include a `distinct_id` with every call on the backend => so that makes the users identified)

> Seems best to not use the word "identified" because that's a logical relation to "$identify", but it's not only that fn usage.

Makes sense

> For folks who want to merge this user with someone they will know how to do it

Good to know

Reply:
> Users on the backend are identified users, no? Is it possible to have anonymous users on the backend? How? (I thought you need to include a distinct_id with every call on the backend => so that makes the users identified)

No. In our code we mark a user `is_identified = true` iff any of their distinct_id's has been used as `distinct_id` for `$identify`, `$create_alias`, `$merge_dangerously`. (note this would be the case after merging https://github.com/PostHog/posthog/pull/15912 currently it's more confusing to know the exact condition 😅 ).
This means that if you are a backend only user and never send any of these events all of your users will have `is_identified = false`. This is because the frontends don't tell us that it's an anonymous ID (there's been talks about sending that info, but historically we don't). Furthermore we want to safeguard agains  aliasing badly everyone together in the backends too.

Reply:
ah okay! Thank you for clearing that up. I've [updated](https://github.com/PostHog/posthog.com/pull/6139) the docs to clarify this.

With this in mind, let's do the following (very similar to what you suggested):
Have `Merge restrictions: none / applied` as you suggested
For the the tooltip: `Can/Cannot be used as alias_id – click for more info.` and have that click point to the alias part of the docs -> https://posthog.com/docs/data/identify#alias-assigning-multiple-distinct-ids-to-the-same-user 

I would still exclude the "Can be used as $anonymous_id", since its not something our users would do, and it would add confusion to include it 

WDYT?

Reply:
Sounds good, will update